Resources/template.json

```diff
@@ -0,0 +1,122 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "appServiceName": {
+            "type": "string",
+            "metadata": {
+                "description": "App Service name to be created"
+            }
+        },
+        "appServicePlanName": {
+            "type": "string",
+            "metadata": {
+                "description": "App Service Plan to put the app service inside"
+            }
+        },
+        "appServicePlanResourceGroup": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
+            "metadata": {
+                "description": "Resource group the App Service Plan is within"
+            }
+        },
+        "appServiceType": {
+            "type": "string",
+            "allowedValues": [
+                "app"
+            ],
+            "defaultValue": "app",
+            "metadata": {
+                "description": "Type of site, either (web)app or functionapp"
```

Comment:
You are not allowing functionapp in the list of allowedValues. If you only ever allow web apps then there is no need for this param